app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/cumulativeTrendControls.jsx

```diff
@@ -126,6 +134,10 @@ export class CumulativeTrendControls extends Component {
     });
   }
 
+  state = {
+    showSecondLevel: false,
```

Comment:
I think that this state property should have a calculated initial value in the case when the user opens a `editWidgetModal` and already has 2 submitted attributes (see the screenshots below).
![image](https://user-images.githubusercontent.com/31861805/62068465-bc659080-b23e-11e9-9efd-9dcb23e8ec38.png)
![image](https://user-images.githubusercontent.com/31861805/62068479-c4bdcb80-b23e-11e9-8764-f8cfc8067336.png)